### PR TITLE
Stop using deprecated UIWebView component

### LINKED ISSUE #5: UIWebView is deprecated
When submitting the app with TalkableSDK framework I am getting this warning from Apple:

ITMS-90809: Deprecated API Usage - App updates that use UIWebView will no longer be accepted as of December 2020. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview).

Found this usage in your repo: https://github.com/talkable/ios-sdk/blob/413eacecba1814195a06f638602f41de356b567f/Talkable/Talkable.m#L827

### PR DESCRIPTION
**Pull request by dgtv**
Fixes #5 
## Related Stories

[](https://talkable.atlassian.net/browse/PR-14161)

### CODE REVIEW

**Comment by zhuravel** [comment]
@dgtv Cool. How’s the resulting user agent looks like?

File: Talkable/Talkable.m [hunk 1]
```diff
@@ -824,9 +824,10 @@ - (NSString*)userAgent {
...
         if (!userAgent) {
-            UIWebView*  webView = [[UIWebView alloc] init];
-            [webView loadHTMLString:@"<html></html>" baseURL:nil];
-            userAgent = [webView stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
+            NSString* systemName = [[UIDevice currentDevice] model];
+            NSString* systemVersion = [[UIDevice currentDevice] systemVersion];
+            systemVersion = [systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
+            userAgent = [NSString stringWithFormat:@"%@ OS %@ webview", systemName, systemVersion];
         }
         if (userAgent) {
```
Comment (zhuravel) [inline Talkable/Talkable.m]: Is this `if` still needed? Looks like the user agent is guaranteed to be present.

**Comment by dgtv** [comment]
> @dgtv Cool. How’s the resulting user agent looks like?

@zhuravel Here is an example: `iPhone OS 11_3 webview;Talkable iOS/1.4.10`.
I've checked it with all our methods in `VisitorUserAgent` and they work as before the change

File: Talkable/Talkable.m [hunk 2]
```diff
@@ -824,14 +824,13 @@ - (NSString*)userAgent {
...
-        }
-        if (userAgent) {
-            NSString* userAgentSuffix = [NSString stringWithFormat:@"Talkable iOS/%@", TKBLVersion];
-            _userAgent = [NSString stringWithFormat:@"%@;%@", userAgent, userAgentSuffix];
+            NSString* systemName = [[UIDevice currentDevice] model];
+            NSString* systemVersion = [[UIDevice currentDevice] systemVersion];
+            systemVersion = [systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
+            userAgent = [NSString stringWithFormat:@"%@ OS %@ webview", systemName, systemVersion];
         }
+        NSString* userAgentSufix = [NSString stringWithFormat:@"Talkable iOS/%@", TKBLVersion];
```
Comment (zhuravel) [inline Talkable/Talkable.m]: You’ve re-introduced the typo, should be Suffix with double F.

**Review by zhuravel — APPROVED** [review]

**Pull request merged**